### DISCUSSION TITLE
Events or webhooks

### DISCUSSION THREAD

**Discussion by GideonStowell** [community/community#40043] [Other Feature Feedback, Questions, & Ideas]
The lists are really great and I've been finding it helpful to categorize things rather than just lump everything together. 

I would really like to see some sort of webhook or events functionality added to the starring and list features. For example, when I star a repo and add it to a specific list, I could call some external api. A common use case could be for tracking student homework repos. I star their repo and it could update my grading software with a link to their repo so I can find it when doing grades. 

I haven't been able to find the answer to if this exists already. I'd love to see some docs if it does. 

Otherwise, I'd be curious if others would find this useful and interesting.

**Comment by ossanna16** [comment]
Hi there @GideonStowell and welcome to our community! Thank you for the feedback!
